modules/mcagar/src/main/ts/ephox/mcagar/api/bdd/TinyState.ts

```diff
@@ -1,14 +1,14 @@
 import { Editor as EditorType } from '../../alien/EditorTypes';
 
 export const withNoneditableRootEditor = <T extends EditorType = EditorType>(editor: T, f: (editor: T) => void): void => {
-  editor.getBody().contentEditable = 'false';
+  editor.setEditableRoot(false);
```

Comment:
I have concerns about switching to `setEditableRoot` without having a fallback. Mcagar is used in other repos and I don't think we want a patch version of mcagar to have to rely on a specific version of TinyMCE, v6.5 in this case

Reply:
The TinyState API would fail only when being executed on pre v6.5 versions not sure why one would call that function on a version that doesn't have that feature.

Reply:
I added a check if that API is available if it's not then an error message about requiring at least 6.5 will be presented.

Reply:
I'm less concerned about this. We tend to use new mcagar on old TinyMCE, not the other way around.